packages-ts/gauntlet-terra/src/commands/internal/terra.ts

```diff
@@ -147,7 +183,7 @@ export default abstract class TerraCommand extends WriteCommand<TransactionRespo
     return this.wrapResponse(res)
   }
 
-  async simulate(signer: AccAddress, msgs: (MsgExecuteContract | MsgSend)[]): Promise<Number> {
+  async simulate(signer: AccAddress, msgs: (MsgExecuteContract | MsgSend | MsgUpdateContractAdmin)[]): Promise<Number> {
```

Comment:
Could we use that parent all encapsulating Msg type here, instead of listing specific messages one by one?

Reply:
Yes. The list is slowly growing so makes sense